bin.src/genInputRegistry.py

```diff
@@ -30,7 +30,8 @@
 import sys
 
 import lsst.daf.base as dafBase
-import lsst.afw.image as afwImage
+from lsst.afw.fits import readMetadata
+from lsst.afw.geom import makeSkyWcs
 import lsst.skypix as skypix
```

Comment:
I hear you and I used to feel that way myself. However, I now strongly prefer to import symbols when only one or two are used, especially if I the names are unambiguous. This allows me to avoid the controversial use of namespace abbreviations or avoid spelling out the full library name prefix every time the symbol is used. I now feel that both alternatives make code harder to read, in that they add characters and often results in extra line wraps. I think this partly comes from getting used to how C++ handles namespaces. In any case it's clearly a matter of taste, and since both are allowed, I'm afraid I'm going to continue importing the symbols. I agree it can get out of hand (afw test_skyWcs.py, alas) and you should feel free to complain if you find instances that have.